<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1584891250077' stop='1584939193926'>
    <name>CheckOut Verfication</name>
    <title>CheckOut Verfication</title>
    <test-cases>
        <test-case start='1584891250077' status='failed' stop='1584891262592'>
            <name>checkout case for One time Address</name>
            <title>checkout case for One time Address</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='a50e90cf-8638-4f86-819f-eae1b3040dfb-attachment.png' type='image/png' size='215058'/>
            </attachments>
            <failure>
                <message>Failed: No element found using locator: By(xpath, //button[@class='btn btn-primary btn-block mb-4 font-weight-bold'])</message>
                <stack-trace>NoSuchElementError: No element found using locator: By(xpath, //button[@class='btn btn-primary btn-block mb-4 font-weight-bold'])
                    at C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\built\element.js:814:27
                    at ManagedPromise.invokeCallback_ (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
                    at C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at processTicksAndRejections (internal/process/task_queues.js:93:5)Error
                    at ElementArrayFinder.applyAction_ (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
                    at ElementArrayFinder.&lt;computed> [as isDisplayed] (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
                    at ElementFinder.&lt;computed> [as isDisplayed] (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
                    at BasePage.click (C:\Users\NareshS\eclipse-workspace\protrract\Mars_Byer\pageobjects\BasePage\BasePage.js:84:22)
                    at addtoCart.clickOnCheckout (C:\Users\NareshS\eclipse-workspace\protrract\Mars_Byer\pageobjects\AddItem\AddItem.js:731:8)
                    at UserContext.&lt;anonymous> (C:\Users\NareshS\eclipse-workspace\protrract\Mars_Byer\specs\Checkout\Checkout_spec.js:39:22)
                    at C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
                    at new ManagedPromise (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
                    at ControlFlow.promise (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
                    at schedulerExecute (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
                From: Task: Run it("checkout case for One time Address") in control flow
                    at UserContext.&lt;anonymous> (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
                    at attempt (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at runNext (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
                    at C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4264:13
                    at C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4172:9
                    at C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:64:48
                    at ControlFlow.emit (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\events.js:62:21)
                    at ControlFlow.shutdown_ (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2674:10)
                    at C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2599:53
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (C:\Users\NareshS\eclipse-workspace\protrract\Mars_Byer\specs\Checkout\Checkout_spec.js:36:6)
                    at addSpecsToSuite (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (C:\Users\NareshS\eclipse-workspace\protrract\Mars_Byer\specs\Checkout\Checkout_spec.js:19:1)
                    at Module._compile (internal/modules/cjs/loader.js:956:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
                    at Module.load (internal/modules/cjs/loader.js:812:32)
                    at Function.Module._load (internal/modules/cjs/loader.js:724:14)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1584891262592' status='failed' stop='1584912886208'>
            <name>checkout process</name>
            <title>checkout process</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.</message>
                <stack-trace>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
                    at Timeout._onTimeout (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4281:23)
                    at listOnTimeout (internal/timers.js:531:17)
                    at processTimers (internal/timers.js:475:7)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1584912886208' status='passed' stop='1584912902034'>
            <name>Verify order has submitted</name>
            <title>Verify order has submitted</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='e0f23e8d-4802-4c99-9505-fc72dfc99907-attachment.png' type='image/png' size='215058'/>
            </attachments>
        </test-case>
        <test-case start='1584912902034' status='failed' stop='1584939187459'>
            <name>Verify submited order Number </name>
            <title>Verify submited order Number </title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='ec3d4c47-0832-4b5f-8478-8289fb7e4906-attachment.png' type='image/png' size='215087'/>
            </attachments>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.</message>
                <stack-trace>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
                    at Timeout._onTimeout (C:\Users\NareshS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4281:23)
                    at listOnTimeout (internal/timers.js:531:17)
                    at processTimers (internal/timers.js:475:7)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1584939187459' status='passed' stop='1584939193920'>
            <name>Order ID Number Text is present when order is confirmed</name>
            <title>Order ID Number Text is present when order is confirmed</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='22f3ad06-06f6-4beb-913b-46799d308661-attachment.png' type='image/png' size='215087'/>
            </attachments>
        </test-case>
        <test-case start='1584939193920' status='pending' stop='1584939193921'>
            <name>Date Submitted Text is present when order is confirmed</name>
            <title>Date Submitted Text is present when order is confirmed</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Temporarily disabled with xit</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
        <test-case start='1584939193921' status='pending' stop='1584939193922'>
            <name> Verify Date Submitted  is the current date when order is confirmed</name>
            <title> Verify Date Submitted  is the current date when order is confirmed</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Temporarily disabled with xit</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
        <test-case start='1584939193922' status='pending' stop='1584939193924'>
            <name>Shipping Method Text is present when order is confirmed</name>
            <title>Shipping Method Text is present when order is confirmed</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Temporarily disabled with xit</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
        <test-case start='1584939193924' status='pending' stop='1584939193925'>
            <name> vrify Shipping Method  is same that user has selected while palcing the order</name>
            <title> vrify Shipping Method  is same that user has selected while palcing the order</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Temporarily disabled with xit</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
        <test-case start='1584939193925' status='pending' stop='1584939193925'>
            <name>Subtotal Text is present when order is confirmed</name>
            <title>Subtotal Text is present when order is confirmed</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Temporarily disabled with xit</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
        <test-case start='1584939193925' status='pending' stop='1584939193926'>
            <name> verify Subtotal value is correct</name>
            <title> verify Subtotal value is correct</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Temporarily disabled with xit</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
        <test-case start='1584939193926' status='pending' stop='1584939193926'>
            <name> verify print option</name>
            <title> verify print option</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Temporarily disabled with xit</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>